netebpfext/net_ebpf_ext.c

```diff
@@ -632,6 +634,46 @@ _net_ebpf_ext_provider_client_attach_callback(
     return EBPF_SUCCESS;
 }
 
+static _Function_class_(KDEFERRED_ROUTINE) _IRQL_requires_max_(DISPATCH_LEVEL) _IRQL_requires_min_(DISPATCH_LEVEL)
+    _IRQL_requires_(DISPATCH_LEVEL) _IRQL_requires_same_ VOID _net_ebpf_ext_rundown(
+        _In_ struct _KDPC* Dpc,
+        _In_opt_ PVOID DeferredContext,
+        _In_opt_ PVOID SystemArgument1,
+        _In_opt_ PVOID SystemArgument2)
```

Comment:
```suggestion
        _In_ struct _KDPC* dpc,
        _In_opt_ void* deferred_context,
        _In_opt_ void* system_argument1,
        _In_opt_ void* system_argument2)
```
use lower_snake_case for argument names just like for variables, and avoid P... types